extensions/amp-story-interactive/0.1/amp-story-interactive-abstract.js

```diff
@@ -814,9 +817,13 @@ export class AmpStoryInteractive extends AMP.BaseElement {
    * @param {boolean} toggle
    */
   toggleTabbableElements_(toggle) {
-    // TODO: Revise tabbable elements on components when considering #31747.
     this.rootEl_.querySelectorAll('button, a').forEach((el) => {
-      el.setAttribute('tabindex', toggle ? 0 : -1);
+      // Disable tabbing through options if already selected.
+      if (this.getOptionElements().includes(el) && this.hasUserSelection_) {
```

Comment:
Why do you need to check if it's an option element? What else could it be?

If you really need to check that, let's test for a class or something. Doing a querySelector + a loop within a loop should be improved

Reply:
The disclaimer icon + learn more link + close button are also buttons, and they should remain tabbable after the user has answered (so they need to be selected). Testing for a class SG